flutter-studio/src/io/flutter/actions/FlutterNewProjectAction.java_stub

```diff
@@ -85,4 +113,139 @@ public class FlutterNewProjectAction extends AnAction implements DumbAware {
     decorated.setIcon(icon, 1, 0, 0);
     return decorated;
   }
+
+  @SuppressWarnings({"UnstableApiUsage", "unchecked"})
+  private void replaceModuleBuilders() {
+    if (true) return;
```

Comment:
I don't understand the early returns in this file - are these replaced for certain versions?